crates/requirements-txt/src/lib.rs

```diff
@@ -486,7 +477,7 @@ impl RequirementsTxt {
                     // _requirements_, but we don't want to support that.
                     for entry in sub_constraints.requirements {
                         match entry.requirement {
-                            RequirementsTxtRequirement::Pep508(requirement) => {
+                            RequirementsTxtRequirement::Uv(requirement) => {
```

Comment:
Why is this changing to support Uv requirements?

Reply:
This trickled down from `RequirementsTxtRequirement` being entangled with `RequirementsSpecification.requirements` (see the other comment), should we change this to use separate types?